src/shared/Navbar/index.tsx

```diff
@@ -1,19 +1,34 @@
 import AppBar from '@material-ui/core/AppBar';
+import { makeStyles } from '@material-ui/core/styles'; 
 import Typography from '@material-ui/core/Typography';
 import Toolbar from '@material-ui/core/Toolbar';
 import React from 'react';
+import CartButton from '../CartButton'
```

Comment:
Just to expand on my suggestion below, I think we could have a src/shared/Navbar/RightAlignedButtons/index.tsx where we import all these buttons, and then in this file we'd only need to import the RightAlignedButtons component.